### PR TITLE
chore(appium-tizen-tv-driver): ungulpify

### PR DESCRIPTION
**Pull request by boneskull**
yeah might as well

### CODE REVIEW

**Comment by boneskull** [comment]
Current dependencies on/for this PR:
* main
  * **PR #36** <a href="https://app.graphite.dev/github/pr/headspinio/appium-tizen-tv-driver/36" target="_blank">Graphite</a>  👈
    * **PR #37** <a href="https://app.graphite.dev/github/pr/headspinio/appium-tizen-tv-driver/37" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/headspinio/appium-tizen-tv-driver/36?utm_source=stack-comment).

File: package.json [hunk 1]
```diff
@@ -48,11 +48,11 @@
     "singleQuote": true
   },
   "dependencies": {
-    "appium-tizen-tv-driver": "file:./packages/appium-tizen-tv-driver"
+    "appium-tizen-tv-driver": "file:./packages/appium-tizen-tv-driver",
+    "tizen-remote": "file:./packages/tizen-remote"
```
Comment (boneskull) [inline package.json:52]: this was missing

Comment (boneskull) [reply package.json:52]: I guess I'm not sure if it actually matters. it used to before npm workspaces.

File: package.json [hunk 2]
```diff
@@ -91,18 +90,7 @@
     "webdriverio": "7.20.9"
   },
   "engines": {
-    "node": ">=8",
-    "npm": ">=6"
-  },
-  "appium": {
```
Comment (boneskull) [inline package.json:97]: this was very confusing for appium's extension detection.

**Review by jlipps — APPROVED** [review]

**Pull request merged**